src/UnityExtension/Assets/Editor/GitHub.Unity/UI/BranchesView.cs

```diff
@@ -304,9 +281,6 @@ private void OnButtonBarGUI()
 
         private void OnTreeGUI(Rect rect)
         {
-            if (!treeLocals.IsInitialized)
-                RefreshBranchList();
```

Comment:
This was ensuring render doesn't get called before initialization happens, so we need to add this check in a different way now we have the cache going